_data/members.yml

```diff
@@ -1,3 +1,6 @@
+- name: Albi Celaj
+ degree: PhD, Molecular Genetics
```

Comment:
Yes, it needs another space. Also, please keep names sorted alphabetically by last name.